zigbee2mqtt/DOCS.md

```diff
@@ -6,6 +6,9 @@ The suggested way to pair your devices is to enable zigbee2mqtt's `permit_join`
 
 You can use [Zigbee2MqttAssistant](https://github.com/yllibed/Zigbee2MqttAssistant) if you don't want to use MQTT directly.
 
+### Enabling the integrated Frontend
+Enable `ingress` to have the frontend available in your UI: **Supervisor → Dashboard → Zigbee2mqtt Edge → Show in sidebar**. You can find more details about the feature on the [Zigbee2mqtt documentation](https://www.zigbee2mqtt.io/information/frontend.html).
```

Comment:
I think you want to remove Edge in this line. I guess this explanation was copied from the z2m edge version.

Reply:
Thanks for spotting this. I will change this after the pipeline finishes and the container is available, as docs can be updated separately without republishing a newer version.

Reply:
Fixed in #429